locales/ja/puppetlabs-inifile.po

```diff
@@ -0,0 +1,76 @@
+# #-#-#-#-#  puppetlabs-inifile.pot (puppetlabs-inifile 2.4.0-7-g89be913)  #-#-#-#-#
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2018 Puppet, Inc.
+# This file is distributed under the same license as the puppetlabs-inifile package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"#-#-#-#-#  puppetlabs-inifile.pot (puppetlabs-inifile 2.4.0-7-g89be913)  #-#-"
+"#-#-#\n"
+"Project-Id-Version: puppetlabs-inifile 2.4.0-7-g89be913\n"
+"\n"
+"Report-Msgid-Bugs-To: docs@puppet.com\n"
+"POT-Creation-Date: 2018-10-02 16:38+0300\n"
+"PO-Revision-Date: 2018-10-02 16:38+0300\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+"#-#-#-#-#  puppet.pot (PACKAGE VERSION)  #-#-#-#-#\n"
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-10-02 16:39:00 +0300\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Translate Toolkit 2.0.0\n"
+
```

Comment:
Just a quick note, but the metadata entry is missing from here.
You can see it in the .pot file below.

```
#. metadata.json
#: .summary
msgid "Resource types for managing settings in INI files"
msgstr ""
```